.github/workflows/ci.yml

```diff
@@ -16,25 +16,84 @@ concurrency:
+        - { runner: "k8s-t2a-standard-16", image: "arm64v8/ros:humble-ros-base-jammy", arch: "ARM64" }
+    name: Build and test bosdyn_msgs packages for ${{ matrix.config.arch }}
+    runs-on: ${{ matrix.config.runner }}
     container:
-      image: ros:humble-ros-base-jammy
+      image: ${{ matrix.config.image }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
         with:
           path: src/bosdyn_msgs
           submodules: true
+      
       - name: Install dependencies
         run: |
           apt update && apt install -y python3-pip
           export PIP_CONSTRAINT=src/bosdyn_msgs/pip-constraint.txt
           rosdep update && rosdep install -i -y --from-path src
-      - name: Build and run tests
+      
+      - name: Build and run tests for ${{ matrix.config.arch }}
         run: |
           source /opt/ros/humble/setup.bash
           colcon build --symlink-install --packages-up-to bosdyn_msgs
           colcon test --event-handlers console_direct+ --packages-up-to bosdyn_msgs
           colcon test-result --all --verbose
+
+  # build_and_test_amd64:
+  #   name: Build and test bosdyn_msgs packages for Amd64
+  #   runs-on:
+  #     group: bdai-gh-large-runners
+  #   container:
+  #     image: ros:humble-ros-base-jammy
+  #   steps:
+  #     - name: Checkout repository
+  #       uses: actions/checkout@v3
+  #       with:
+  #         path: src/bosdyn_msgs
+  #         submodules: true
+      
+  #     - name: Install dependencies
+  #       run: |
+  #         apt update && apt install -y python3-pip
+  #         export PIP_CONSTRAINT=src/bosdyn_msgs/pip-constraint.txt
+  #         rosdep update && rosdep install -i -y --from-path src
+      
+  #     - name: Build and run tests
+  #       run: |
+  #         source /opt/ros/humble/setup.bash
+  #         colcon build --symlink-install --packages-up-to bosdyn_msgs
+  #         colcon test --event-handlers console_direct+ --packages-up-to bosdyn_msgs
+  #         colcon test-result --all --verbose
+
+  # build_and_test_arm64:
+  #   name: Build and test bosdyn_msgs packags for Arm64
+  #   runs-on:
+  #     group: k8s-t2a-standard-16
+  #   contianer:
+  #     image: arm64v8/ros:humble-ros-base-jammy
+  #   steps:
+  #     - name: Checkout repository
+  #       uses: actions/checkout@v3
+  #       with:
+  #         path: src/bosdyn_msgs
+  #          submodules: true
+
+  #     - name: Install dependencies
+  #       run: |
+  #         apt update && apt install -y python3-pip
+  #         export PIP_CONSTRAINT=src/bosdyn_msgs/pip-constraint.txt
+  #         rosdep update && rosdep install -i -y --from-path src
+
+  #     - name: Build and run tests
+  #       run: |
+  #         source /opt/ros/humble/setup.bash
+  #         colcon build --symlink-install --packages-up-to bosdyn_msgs
+  #         colcon test --event-handlers console_direct+ --packages-up-to bosdyn_msgs
+  #         colcon test-result --all --verbose
```

Comment:
@amessing-bdai nit: let's just remove commented sections. We can bring them back later on if need be.

Reply:
That was the plan. They were just commented out while I was trying something else